### PR TITLE
Add documentation for octopusdeploy_deploy module

### PR DESCRIPTION
**Pull request by james-otten**
Add documentation for ```exploit/windows/http/octopusdeploy_deploy```

## Verification

- [x] **Verify** provided documentation is reasonably coherent.

### CODE REVIEW

File: documentation/modules/exploit/windows/http/octopusdeploy_deploy.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+  **USERNAME**
+
+  Username of the Octopus Deploy user.
+
+  **PASSWORD**
+
+  Password of the Octopus Deploy user.
+
+  **RHOST**
```
Comment (h00die) [inline documentation/modules/exploit/windows/http/octopusdeploy_deploy.md]: This should be self explanatory and not needed

File: documentation/modules/exploit/windows/http/octopusdeploy_deploy.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+  Host running Octopus Deploy server.
+
+  **RPORT**
```
Comment (h00die) [inline documentation/modules/exploit/windows/http/octopusdeploy_deploy.md]: This should be self explanatory and not needed

**Comment by h00die** [comment]
two minor things to shorten the docs up, these look great though!  If you take out those two self explanatory sections, i should be able to approve this tomorrow

**Comment by james-otten** [comment]
Thanks for the feedback @h00die. I've made the requested changes.

**Comment by h00die** [comment]
# Release Notes
Adds docs for octopusdeploy_deploy

**Comment by h00die** [comment]
@james-otten Thanks for the submission, nice docs!  Made that very easy.

**Pull request merged**